test/bravery-components/braveryPanelTest.js

```diff
@@ -895,6 +895,20 @@ describe('Bravery Panel', function () {
         .tabByIndex(0)
         .waitUntil(verifyProxyBlocking)
     })
+    it('blocking access to fingerprinting methods on iframe.contentWindow', function * () {
+      const url = Brave.server.url('fingerprinting-blocking-from-child-frames.html')
+      yield this.app.client
+        .tabByIndex(0)
+        .loadUrl(url)
+        .waitForUrl(url)
+        .openBraveMenu(braveMenu, braveryPanel)
+      yield changeFpSetting(this.app.client, blockFpOption)
+      yield this.app.client
+        .waitForTextValue(fpStat, '1')
```

Comment:
Should this be removed for Linux? The test looks to be failing due to this.

https://travis-ci.org/brave/browser-laptop/jobs/303759640#L5789

````
  3) Bravery Panel Adblock stats without iframe tests blocking access to fingerprinting methods on iframe.contentWindow:
     Promise was rejected with the following reason: timeout
  Error
````

Reply:
i think this should pass on linux in general